sonic_platform_base/sonic_xcvr/fields/consts.py

```diff
@@ -32,6 +32,11 @@
 RX_POWER_HIGH_WARNING_FIELD = "RxPowerHighWarning"
 RX_POWER_LOW_WARNING_FIELD = "RxPowerLowWarning"
 
+RX_FAULT_FIELD = "RxFault"
+RX_FAULT_SUPPORT_FIELD = "RxFaultSupported"
```

Comment:
@vdahiya12 Seems we are not using these currently. Am I missing something?

Reply:
removed, the RX_FAULT_SUPPORT_FIELD is also not used but i still added it for later uses.